Makefile

```diff
@@ -254,6 +267,13 @@ deploy/integreatly-rhmi-cr.yml:
 	sed "s/SELF_SIGNED_CERTS/$(SELF_SIGNED_CERTS)/g" | \
 	sed "s/OPERATORS_IN_PRODUCT_NAMESPACE/$(OPERATORS_IN_PRODUCT_NAMESPACE)/g" | \
 	sed "s/USE_CLUSTER_STORAGE/$(USE_CLUSTER_STORAGE)/g" > deploy/integreatly-rhmi-cr.yml
+ifneq ( ,$(findstring image_mirror_mapping,$(IMAGE_MAPPINGS)))
```

Comment:
Not sure about the use of `''`. I had to put it in for this sed to work. Would be good to test this on a linux machine. I'm running mac.

Reply:
The `sed -i '' ...` doesn't work for linux, the only  way to have sed compatible on mac and linux it to do so `sed -i.bak ...` but this will create an additional file, otherwise the alternative is to read and write in the same file using the stdout

Reply:
@b1zzu thanks for the info. would you mind making the change locally for now and I will figure a way to update this.

Reply:
yes I will try it locally for now

Reply:
I found out this two options:
```
sed "s/DELOREAN_PULL_SECRET_NAMESPACE/$(NAMESPACE)/g" deploy/integreatly-rhmi-cr.yml | \
sed "s/DELOREAN_PULL_SECRET_NAME/$(DELOREAN_PULL_SECRET_NAME)/g" > deploy/integreatly-rhmi-cr.yml.new
mv deploy/integreatly-rhmi-cr.yml{.new,} 
```
or 
```
sed -i.bak "s/DELOREAN_PULL_SECRET_NAMESPACE/$(NAMESPACE)/g" deploy/integreatly-rhmi-cr.yml
sed -i.bak "s/DELOREAN_PULL_SECRET_NAME/$(DELOREAN_PULL_SECRET_NAME)/g" deploy/integreatly-rhmi-cr.yml
rm deploy/integreatly-rhmi-cr.yml.bak
```
you can try them on Mac and which of the two works for you

Reply:
The second one works. Pushed it up. Resolving this comment. Thanks for the code. 👍